app/src/components/App.jsx

```diff
@@ -15,6 +15,20 @@ class App extends Component {
       }
     }
     this.props.getLoggedUser();
+
+    if (!DISABLE_WELCOME_MODAL) this.props.setWelcomeModalUrl();
```

Comment:
I would highly suggest using explicit comparisons in that kind of situation, ie
```
DISABLE_WELCOME_MODAL !== true
```
You always get taken by surprise when smth is falsy or truthy unexpectedly.